frontend/src/pages/home/components/NewMeeting.tsx

```diff
@@ -171,167 +70,24 @@ export default function NewMeeting({
-                                        {[...Array(5).keys()]
-                                            .map((hour) =>
-                                                [0, 15, 30, 45].map((minute) => (
-                                                    <option key={`${hour}-${minute}`} value={`${hour}:${minute}`}>
-                                                        {hour}h {minute}m
-                                                    </option>
-                                                )),
-                                            )
-                                            .flat()}
-                                    </select>
-                                </div>
-                            </div>
-                        </div>
-
-                        {/* magic */}
-                        {/* <AutocompleteInput
-                            options={roles.roles.filter((role) => role.name !== 'Admin' && role.name !== 'Default')}
-                            label="Which group of people are you inviting?"
-                            value={selectedRoles}
-                            onChange={setSelectedRoles}
-                        /> */}
-
-                        {/* Description */}
-                        <label className="block text-gray-700 mb-1 w-full" htmlFor="description">
-                            Description
-                        </label>
-                        <div className="relative w-full my-2">
-                            <textarea
-                                id="description"
-                                className="focus:outline-none rounded-md w-full py-2 pl-3 pr-10 transition-colors duration-200 bg-gray-50 focus:bg-gray-100"
-                                name="description"
-                                placeholder="Enter description"
-                                rows={5}
-                                value={formValues.description}
-                                onChange={handleInputChange}
-                            />
-                        </div>
-
-                        {/* Submit Button */}
-                        <button
-                            className="bg-[#7d6ca3] hover:bg-opacity-90 text-white py-2 px-4 rounded-md text-xl mt-4 transition-transform duration-200 transform hover:scale-95"
-                            type="submit"
-                        >
-                            {isEditMeeting ? 'Edit meeting detail' : 'Create new meeting'}
-                        </button>
-                    </form>
-                </div>
-            </div>
-        )
+        <>
+            {/* Force refresh of modal */}
+            {isNewMeetingOpen && (
+                <MeetingModal
+                    isOpen={isNewMeetingOpen}
+                    onSubmit={handleSubmit}
+                    setIsOpen={setIsNewMeetingOpen}
+                    users={users ? users.users : []}
+                    isCreate
```

Comment:
isCreate is never defined in NewMeeting.tsx, which if you pass it as a prop wouldn't it be undefined which idk if that is how it's intended? Would you care to explain this to me lol

Reply:
If you pass prop like that it just passes in true, otherwise it will just be undefined
Sorta like how input fields have `required` and there isn't a need to say required={true}